webpack/src/ForemanDiscovery/DiscoveredHosts/Components/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ForemanEmptyState from "foremanReact/components/common/EmptyState";
+
+const EmptyState = (props) => {
+  const description =
+    "No discovered hosts found in this context. This page shows discovered bare-metal or virtual nodes waiting to be provisioned.";
+  const documentation = {
+    url: props.documentation,
+  };
```

Comment:
can you pass the `documentation` object fixed from the server?

Reply:
I am not sure, if I follow this comment. Can you please help with it?

Reply:
Can you rename the props from `documentation` to `docUrl`?